tests/components/eafm/conftest.py

```diff
@@ -0,0 +1,48 @@
+"""eafm fixtures."""
+# pylint: disable=redefined-outer-name
+
+import asyncio
+import datetime
+from unittest import mock
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.fixture()
+def mock_get_stations():
+    """Mock aioeafm.get_stations."""
+    with patch("homeassistant.components.eafm.config_flow.get_stations") as patched:
+
+        def set_result(value):
+            future = patched.return_value = asyncio.Future()
+            if isinstance(value, Exception):
+                future.set_exception(value)
+            else:
+                future.set_result(value)
+
+        yield set_result
+
+
+@pytest.fixture()
+def mock_get_station():
+    """Mock aioeafm.get_station."""
+    with patch("homeassistant.components.eafm.station.get_station") as patched:
+
+        def set_result(value):
+            future = patched.return_value = asyncio.Future()
```

Comment:
Can we simplify this fixture by using and importing patch from asynctest?

Reply:
Did this